homeassistant/components/sensor/forecast.py

```diff
@@ -134,11 +137,20 @@ def update(self):
         import forecastio
 
         self.forecast_client.update()
-        data = self.forecast_client.data
+        data = self.forecast_client.data.currently()
+        data_minutely = self.forecast_client.data.minutely()
```

Comment:
According to the [python-forecastio docs](https://github.com/ZeevG/python-forecast.io#function-forecastioload_forecastkey-latitude-longitude) when we call forecastio.load_forecast() we get all the data, so it shouldn't be any extra requests.